tests/functional/spec/features/calculated_summary/new_calculated_summary_repeating_and_static_answers.spec.js

```diff
@@ -13,12 +13,9 @@ import SupermarketTransportPage from "../../../generated_pages/new_calculated_su
 import SupermarketTransportCostPage from "../../../generated_pages/new_calculated_summary_repeating_and_static_answers/supermarket-transport-cost.page";
 import CalculatedSummaryPipingPage from "../../../generated_pages/new_calculated_summary_repeating_and_static_answers/calculated-summary-piping.page";
 import { assertSummaryValues, click } from "../../../helpers";
+import { expect } from "@wdio/globals";
```

Comment:
It is in the yarn.lock deps. It is not installed as a direct dependency which is why there is a warning in the IDE but I didn't add it since once we migrate all to built-in expect, the import statement won't be needed. Can add if we think it should be until it has been fully migrated.

Reply:
Ahh - yep makes sense 👍